### DISCUSSION TITLE
怎么格式化代码 ?

### DISCUSSION THREAD

**Discussion by xiaobinqt** [shuzijun/leetcode-editor#562] [Q&A] [answered]
代码直接就顶格了，:point_down:

![image](https://user-images.githubusercontent.com/23259661/187111340-ea888a1e-0f5d-4460-b731-17570fec44eb.png)

请问下可以格式化代码吗 ?

**Comment by shuzijun** [comment, marked as answer]
生成代码之后，可以使用ide的格式化代码功能
